docs/source/plotting.rst

```diff
@@ -0,0 +1,426 @@
+Plotting
+========
+
+The plotting interface on streamz DataFrame and Series objects
+attempts to mirror the pandas plotting API, but instead of plotting
+with matplotlib_ uses HoloViews_ to generate dynamically streaming
```

Comment:
Do we need an a pronoun? "with matplotlib *it* uses HoloViews".

Reply:
Yes, reads better that way.